app/urls.py

```diff
@@ -0,0 +1,24 @@
+"""
+URL configuration for app project.
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/5.1/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
```

Comment:
The 'app_name' variable is correctly set to 'cinema', which is necessary for namespacing the URLs in the Django app.